examples/pokemon-service-client-usage/examples/handling-errors.rs

```diff
@@ -77,10 +77,6 @@ async fn main() {
                 GetStorageError::ValidationError(ve) => {
                     tracing::error!(error = %ve, "A required field has not been set.");
                 }
-                // An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
-                GetStorageError::Unhandled(uh) => {
-                    tracing::error!(error = %uh, "An unhandled error has occurred.")
-                }
```

Comment:
if this is an example it should probably use `DisplayErrorContext`

Reply:
Good call. Fixed.